EventsExpress/Controllers/NotificationTypeController.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace EventsExpress.Controllers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using AutoMapper;
+    using EventsExpress.Core.DTOs;
+    using EventsExpress.Core.IServices;
+    using EventsExpress.ViewModels;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+
+    [Route("api/[controller]")]
+    [Authorize(Roles = "Admin")]
```

Comment:
We don't allow it only for 'Admin'. Consider doing only the Authorize attribute here.